src/command/cmd_defs.c

```diff
@@ -1283,8 +1283,8 @@ static struct cmd_t command_defs[] =
             { "chat off",                  "Do not show time in chat windows." },
             { "muc set <format>",          "Set time format for chat room windows." },
             { "muc off",                   "Do not show time in chat room windows." },
-            { "mucconfig set <format>",    "Set time format for chat room config windows." },
-            { "mucconfig off",             "Do not show time in chat room config windows." },
+            { "config set <format>",       "Set time format for chat room config windows." },
```

Comment:
`chat room config windows` is not accurate anymore. Since now the config window is used for muc config and for your ad-hoc.

Reply:
done